index.js

```diff
@@ -77,12 +79,21 @@ class TimeoutError extends Error {
 	}
 }
 
+let abortController;
```

Comment:
A global AbortController for all requests? That seems wrong. It should be one controller per request